### ISSUE TITLE
Support GHC 9.2

### ISSUE THREAD

**Issue by chshersh** [co-log/co-log#230]

**Comment by chshersh** [comment]
Currently blocked by `chronos`:

* https://github.com/andrewthad/chronos/issues/70

**Comment by brprice** [comment]
> Currently blocked by `chronos`:
> 
>     * [Support GHC 9.2 andrewthad/chronos#70](https://github.com/andrewthad/chronos/issues/70)

Although that issue is still open, the recent hackage revision to [chronos-1.1.4](https://hackage.haskell.org/package/chronos-1.1.4/revisions/) bumped `base` bounds and so supports ghc 9.2. I have a (seemingly) working `co-log` with ghc 9.2.4 by using
```
allow-newer: co-log-0.4.0.1:base
           , co-log-0.4.0.1:bytestring
           , co-log-0.4.0.1:co-log-core
           , co-log-0.4.0.1:typerep-map
```
so I hope that this now just needs some version bumps.

**Comment by vrom911** [comment]
Closed in #248
